### PR TITLE
Add specifications

### PR DESCRIPTION
**Pull request by julkue**
A continuously updated PR to complete the specifications. We can add as much commits as we need, I'll merge them using a squash commit at the end.

This should not replace the concept & specification issue. Everything about this specification should still be discussed there. But we need to present the result of this discussion inside a documentation including history (commits).

### CODE REVIEW

File: spec/language-file.md [hunk 1]
```diff
@@ -64,31 +65,33 @@ A language file is used to collect diacritics mapping information for each langu
 
 ## metadata.alphabet
 
-Required
```
Comment (julkue) [inline spec/language-file.md]: Is there a reason why you've removed the blanks necessary for a line break?

File: spec/language-file.md [hunk 2]
```diff
@@ -98,25 +101,62 @@ An optional object containing two properties:
...
+
+This is the character, or combination of characters used to represent the
+diacritic character (e.g. `�` decomposes into `ue` in German).
+
+### equivalents
+
+Optional
+Type: `Array`
+
+Equivalents are contained within the [[equivalents file]] and are concatenated
```
Comment (julkue) [inline spec/language-file.md]: Did you wanted to create a link here?

Comment (Mottie) [reply spec/language-file.md]: Oh, oops... that's how you link to other files in the same folder when using the GitHub wiki.

Comment (julkue) [reply spec/language-file.md]: Don't worry, take [this](https://github.com/julmot/diacritics/pull/3/commits/b9ed0d1d57cb286c170a64d7803859832e842d70#diff-e910483371fc42dbf4ebf8ec94d954e6R10) as a reference for links used multiple times.

File: spec/language-file.md (hunk 1 above)
Comment (Mottie) [reply spec/language-file.md]: Accidental.

**Comment by julkue** [comment]
@Mottie It's now time to merge this branch. I've reviewed everything and it seems to be finished. May I ask you to take a final look at it, also with regard to spelling mistakes?

**Review by Mottie — APPROVED** [review]
Everything looks good to me. I think it's ready for merging.

**Pull request merged**